workflow/rules/filtering.smk

```diff
@@ -0,0 +1,52 @@
+rule samtools_view:
+    input:
+        "results/merged/dedup/merged_dedup_{sample}.bam"
```

Comment:
Is the doubling of info in `/merged/dedup/` and `merged_dedup_` so that multiqc can tell the difference between the different stats files? If there was any way around that doubling, this would look cleaner, but I guess multiqc only includes the basename of the file, not its path? And simply putting all bams in one folder would create a very messy folder... I have one suggestion I'll make below at the `stats.smk` file, let me know what you think.